Gemfile

```diff
@@ -72,6 +71,7 @@ end
 group :development do
   # Use console on exceptions pages [https://github.com/rails/web-console]
   gem "web-console"
+  gem "annotate", "~> 3.2"
```

Comment:
This is a bonus gem. It adds comments to the top of model files so that you don't need to read the schema.rb file to see the columns of a model.